frontend/src/lib/components/PropertyFilters/components/TaxonomicPropertyFilter.tsx

```diff
@@ -98,8 +100,14 @@ export function TaxonomicPropertyFilter({
                         {index === 0 ? (
                             <>
                                 <span className="arrow">&#8627;</span>
-                                <span className="text">where</span>
+                                <span className="primary-alt">
+                                    <b>where</b>
+                                </span>
                             </>
+                        ) : propertyGroupType ? (
+                            <div className="primary-alt">
```

Comment:
This primary-alt color looks kind of out of place here (like, similar to black, but subtly off) and as far as I've seen it's not part of the design.

<img width="75" alt="Screen Shot 2022-02-18 at 15 31 41" src="https://user-images.githubusercontent.com/4550621/154701937-f8aeada4-49f2-4f4e-944b-32fcf8885ced.png">